tsconfig.json

```diff
@@ -11,8 +11,10 @@
     // "disableReferencedProjectLoad": true,             /* Reduce the number of projects loaded automatically by TypeScript. */
 
     /* Language and Environment */
-    "target": "ESNext",                                  /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
-    "lib": ["esnext", "dom", "dom.iterable"],                                       /* Specify a set of bundled library declaration files that describe the target runtime environment. */
+    "target": "ESNext" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
```

Comment:
I guess esnext maybe too new?

Reply:
I believe this is not an issue. For CJS and ESM, the default is `esnext`, and the target is ultimately determined by the developer's compiled project; UMD uses `es2017`.